inventory/src/main/java/bisq/inventory/InventoryWebServer.java

```diff
@@ -484,14 +487,12 @@ private String getErrorMsgLine(NodeAddress seedNode,
                 RequestInfo requestInfo1 = requestInfoList.get(i);
 
                 // We ignore old errors as at startup timeouts are expected and each node restarts once a day
-                if (requestInfo1.getRequestStartTime() > 0 &&
-                        System.currentTimeMillis() - requestInfo1.getRequestStartTime() > TimeUnit.HOURS.toMillis(24)) {
```

Comment:
I almost left a comment regarding a little precedence confusion, re:

    ( System.currentTimeMillis() - requestInfo1.getRequestStartTime()  ) > TimeUnit.HOURS.toMillis(24)

The precedence rules work, but now it's clear from the change below.